File: app/src/main/java/com/frankly/randomdog/networkresponse/GenerateDogResponse.java
package com.frankly.randomdog.networkresponse;

public class GenerateDogResponse {

    private String message;
    private String status;

    public GenerateDogResponse(String message, String status) {
        this.message = message;
        this.status = status;
    }

    public String getMessage() {
        return message;
    }

    public String getStatus() {
        return status;
    }
}

File: app/src/main/java/com/frankly/randomdog/networkresponse/GenerateDogResponse.kt
package com.frankly.randomdog.networkresponse

/**
 * @author: ArshVardhanAtreya <arshvardhan></arshvardhan>@yahoo.co.in>
 */
class GenerateDogResponse(private val message: String?, private val status: String?) {
    fun getMessage(): String? {
        return message
    }

    fun getStatus(): String? {
        return status
    }

}